### ISSUE TITLE
Dashes in event titles are missing

### ISSUE THREAD

**Issue by ellenli** [ellenli/sunrise#3]
![image](https://cloud.githubusercontent.com/assets/933419/21820093/52655100-d73c-11e6-994c-1e60f6100e3e.png)

Gotta make [sunrise.js](https://github.com/ellenli/sunrise/blob/master/extension/sunrise.js):31 more specific, because

>       eventHTML.innerHTML = eventHTML.innerHTML.replace(/-/g,'');

removes all dashes, including the ones in the event title.

**Comment by AlexeiDarmin** [comment]
Instead of `eventHTML.innerHTML = eventHTML.innerHTML.replace(/-/g,'');`

```
let title     = eventHTML.innerHTML
let dashIndex = title.indexOf('-')

eventHTML.innterHTML = title.substr(0, dashIndex) + title.substr(dashIndex + 1, title.length)
```

^Takes everything except the first dash

**Comment by ellenli** [comment]
Fixed in afa8e5b71c74304caf29e7f5abcd20a1e0735cbd by removing `g` (global flag) from `replace()` lol

![image](https://cloud.githubusercontent.com/assets/933419/21876468/47aeb826-d851-11e6-8ff3-c1e73b3ff015.png)
